src/ui/map.js

```diff
@@ -1143,11 +1143,16 @@ class Map extends Camera {
      */
     getProjection(): ProjectionSpecification {
         if (this._explicitProjection) {
-            return this._explicitProjection;
+            if (this.transform.mercatorFromTransition) {
+                return {name: "globe", center: [0, 0]};
+            }
+            return this.transform.getProjection();
```

Comment:
If we're keeping `_explicitProjection` as a projection object (not a boolean),  can the simpler` return this._explicitProjection;` work here?